### PR TITLE
*: update some docs

### LINKED ISSUE #4315: Format of the commit message needs to update
We have enabled the commit message checker to ensure each PR has the corresponding issue. We need to update https://github.com/tikv/pd/blob/master/CONTRIBUTING.md#format-of-the-commit-message

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

close #4315

### What is changed and how it works?

This PR updates some stale information in our docs.

### Check List

Tests

- No code

### Release note

```release-note
None
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

**Review by HunDunDM — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -57,6 +58,8 @@ The format can be described more formally as follows:
 <BLANK LINE>
 <why this change was made>
 <BLANK LINE>
+<close/fix/ref/resolve> <issue number>
```
Comment (disksing) [inline CONTRIBUTING.md:61]:
```suggestion
<close/fix/ref/resolve> #<issue number>
```

Comment (rleungx) [reply CONTRIBUTING.md]: done

**Review by disksing — APPROVED** [review]

**Pull request merged**